eng/common/pipelines/templates/steps/publish-artifact.yml

```diff
@@ -10,18 +10,25 @@ parameters:
   ArtifactName: ''
   ArtifactPath: ''
   CustomCondition: true
+  DisplayName: ''
 
 steps:
-  - task: PublishPipelineArtifact@1
+  - task: 1ES.PublishPipelineArtifact@1
     condition: and(succeeded(), ${{ parameters.CustomCondition }})
-    displayName: 'Publish ${{ parameters.ArtifactName }} Artifacts'
+    ${{ if parameters.DisplayName }}:
```

Comment:
Is this necessary? Do we really need a custom display name? Or perhaps we can set the DisplayName to a default value in the parameter definition to avoid the need for an extra condition here.

Reply:
I was finding several places where we were doing publishes in the pipeline that had descriptive names that I thought would be useful to preserve. The conflict is preserving the "failed" description, which I also think was useful. I think we have to pick one or the other.

Reply:
@weshaggard updated a bit to remove the conditional